models/ed/R/met2model.ED2.R

```diff
@@ -114,14 +114,25 @@ met2model.ED2 <- function(in.path, in.prefix, outfolder, start_date, end_date, l
     sec  <- nc$dim$time$vals
     Tair <- ncdf4::ncvar_get(nc, "air_temperature")
     Qair <- ncdf4::ncvar_get(nc, "specific_humidity")  #humidity (kg/kg)
-    U    <- ncdf4::ncvar_get(nc, "eastward_wind")
-    V    <- ncdf4::ncvar_get(nc, "northward_wind")
+    U    <- try(ncdf4::ncvar_get(nc, "eastward_wind"),  silent = TRUE)
+    V    <- try(ncdf4::ncvar_get(nc, "northward_wind"), silent = TRUE)
```

Comment:
No need to change, but I recently figured out how to do this with `tryCatch`. In general, it looks something like this:

```r
U <- tryCatch(
    ncdf4::ncvar_get(...),
    error = function(e) {
        logger.info("Wind is absent, so returning NULL")
        return(NULL)
    }
)
```

Here, it's a more complicated nested check, so your approach is probably better, but just wanted to point this out.